engine/flink/executor/src/main/scala/pl/touk/nussknacker/engine/process/registrar/FlinkProcessRegistrar.scala

```diff
@@ -174,7 +177,7 @@ class FlinkProcessRegistrar(compileProcess: (EspProcess, ProcessVersion, Deploym
       val contextTypeInformation = typeInformationDetection.forContext(part.validationContext)
 
       val start = source
-        .sourceStream(env, nodeContext(part.id, Left(ValidationContext.empty)))
+        .sourceStream(env, nodeContext(NodeComponentInfo(part.id, "source", ComponentType.Source), Left(ValidationContext.empty)))
```

Comment:
Why it is hardcoded name. We should hardcode it only for base component types. I wondering if we can do it more explicitly to someone not copy this code. Can u add `NodeComponentInfo.forBaseNode(nodeId, componentType)` (without name)?